src/work-allocation-2/containers/task-assignment/task-assignment-container.component.ts

```diff
@@ -31,23 +32,29 @@ export class TaskAssignmentContainerComponent implements OnInit {
   public excludedCaseworkers: Caseworker[];
   public location: Location;
 
+  public formGroup: FormGroup;
+
+  private assignTask$: Subscription;
```

Comment:
if this is a subscription not Observable it doesn't need a $ from the naming convention, ideally only Observable needs a $ when it's used in pipe or subscribed.